src/server/system_services/account_server.js

```diff
@@ -1192,6 +1204,7 @@ function get_account_info(account, include_connection_cache) {
             endpoint: credentials.endpoint,
             identity: credentials.access_key,
             auth_method: credentials.auth_method,
+            azure_log_access_keys: credentials.azure_log_access_keys,
```

Comment:
if we consider it as sensitive then we shouldn't return it here (same as secret_key)